backend/user_management/ai/llm.py

```
@@ -0,0 +1,46 @@
+            return "Invalid order command. Use 'Select {product_id} {size}' to place an order."
+
+    # Otherwise, continue conversation with OpenAI
+    try:
+        response = openai.ChatCompletion.create(
+            engine=DEPLOYMENT_NAME,
+            messages=messages,
+            temperature=0.7
+        )
+        return response["choices"][0]["message"]["content"].strip()
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)